src/checkErrors.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable max-len */
+const errors = [
+  {
+    message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  },
+  {
+    message: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  },
+  {
+    message: 'This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+  },
+];
+
+const checkErrors = (url, toCase) => {
+  const errorMessage = [];
+
+  if (!url.length) {
+    errorMessage.push(errors[0]);
```

Comment:
it's not stable solution with getting an errors by index
will be better to create separate constants for each of them